### PR TITLE
Fixed main page display style from flex to block

### PR DESCRIPTION
**Pull request by IvayloSlavchev**

### CODE REVIEW

**Review by petya0111 — COMMENTED** [review]
I think navigation left sidebar should move alongside content of the page also.

**Comment by ZahariCheyrekov** [comment]
Great work, @petya0111! I like the overall appearence of the login and register pages. You can set body `margin: 0` and `margin: 2rem;` to the form wrapper element so the login/register form is correctly positioned on the smaller devices.

File: client/src/components/Styles/Header.scss [hunk 1]
```diff
@@ -46,6 +45,22 @@
     }
 }
 
+@media only screen and (min-width: 600px) and (max-width: 900px) {
```
Comment (cholakovvv) [inline client/src/components/Styles/Header.scss]: Try to made the responsive design by using MUI Breakpoints. https://mui.com/material-ui/customization/breakpoints/

File: client/src/components/Styles/NavigationBar.scss [hunk 2]
```diff
@@ -47,30 +49,33 @@
...
-
-        border-radius: 0 25px 0 0;
+        width: 25px;
+        border-top-right-radius: 25px;
         box-shadow: 0 -25px 0 0 rgb(235, 233, 233);
-
+        right: 0;
+    }
+}
+@media only screen and (min-width: 600px) and (max-width: 900px) {
```
Comment (cholakovvv) [inline client/src/components/Styles/NavigationBar.scss]: Same here

File: client/src/components/Styles/Stories.scss [hunk 3]
```diff
@@ -1,143 +1,132 @@
...
-    .stories-root-box {
-        .stories-paragraph {
-            font-size: 2.2rem;
-        }
 
-        .story-data-box {
-            height: 75vh;
+}
 
+@media only screen and (min-width: 700px) and (max-width: 900px) {
```
Comment (cholakovvv) [inline client/src/components/Styles/Stories.scss]: Here too

File: client/src/components/Styles/Stories.scss [hunk 4]
```diff
@@ -1,143 +1,132 @@
...
-    .stories-root-box {
-        .stories-paragraph {
-            font-size: 2.5rem;
-        }
-        
-        .story-data-box {
-            height: 76vh;
-        
-
+@media only screen and (min-width: 600px) and (max-width: 699px){
```
Comment (cholakovvv) [inline client/src/components/Styles/Stories.scss]: Here too

**Review by cholakovvv — COMMENTED** [review]
Good work Ivo, but you can try MUI Breakpoints

**Review by petya0111 — APPROVED** [review]
I hope to handle all with this bubble logic.

**Review by petya0111 — COMMENTED** [review]
On some resolutions there are inconsistencies
image
image
image
image

**Review by petya0111 — APPROVED** [review]
Perfect work and challenge with this responsive view ♥ 'закова го!'

**Review by cholakovvv — APPROVED** [review]
Great work!

**Pull request merged**